### PR TITLE
hostNetwork and initial pass/fail

### PR DESCRIPTION
**Pull request by jtaleric**
- added to the SUT function to deploy hostNetwork
- added initial work for pass/fail criteria

pass/fail is currently only for tcp_stream, and we must have hostNetwork data to compare the podNetwork to.

Signed-off-by: Joe Talerico (rook) <joe.talerico@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,54 +28,89 @@ Ensure your `kubeconfig` is properly set to the cluster you would like to run `k
...
+   duration: 3             # How long to run the test
+   samples: 1              # Iterations to run specified test
+   messagesize: 1024       # Size of the data-gram
+   service: false          # If we should test with the server pod behind a service
+```
+
+## Pass / Fail
+`k8s-netperf` has a cli option for `-tcp-tolerance` which defaults to 10%.
+
+In order to have `k8s-netperf` determine pass/fail `k8s-netperf` will determine if there is enough context to determine pass or fail. Today, this means `k8s-netperf` must be able to run with hostNetwork and podNetwork across nodes -- meaning with `-local` we cannot determine pass/fail.
```
Comment (dry923) [inline README.md:58]: can we cut down on the word determine in the first sentence? you used it 3 times lol

File: containers/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-RUN dnf install -y --nodocs make --enablerepo=centos9 --allowerasing
-RUN dnf install -y --nodocs gcc --enablerepo=centos9 --allowerasing
-RUN curl -LO https://github.com/HewlettPackard/netperf/archive/refs/tags/netperf-2.7.0.tar.gz
-RUN tar -xzf netperf-2.7.0.tar.gz
-WORKDIR /netperf-netperf-2.7.0
-RUN ./configure && make && make install
-RUN rm -rf /netperf-2.7.0.tar.gz /netperf-netperf-2.7.0
-RUN dnf remove -y gcc make
\ No newline at end of file
+RUN dnf install -y --nodocs make automake --enablerepo=centos9 --allowerasing
```
Comment (rsevilla87) [inline containers/Dockerfile:4]: I think netperf is available in EPEL, maybe it's worth to use it rather than compiling?

Comment (jtaleric) [reply containers/Dockerfile:4]: It is quite old -- there are some recent fixes that are not in the 2.7 builds :(

**Comment by jtaleric** [comment]
Self comment to make hostNet optional and mention if a failure happens check Security groups.

File: README.md (hunk 1 above)
Comment (jtaleric) [reply README.md]: @dry923 bettter??

**Pull request merged**